src/utils/global.ts

```diff
@@ -458,7 +459,14 @@ export class GlobalState {
 		)
 			status = GlobalStatus.Unregistered;
 		else if (this.troubleConnecting) status = GlobalStatus.Reconnecting;
-		else status = GlobalStatus.Connected;
+		else {
+			status = GlobalStatus.Connected;
+			posthog.identify(`user:${global.currentIdentity.identityId}`, {
+				name: global.currentIdentity.displayName,
+				email: global.currentIdentity.linkedAccounts[0].email.email,
```

Comment:
Needs to handle no linked accounts for guests.

Reply:
What happens if the user goes from guest -> registered, does this update the user?